backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/submission/SubmissionRecord.java

```diff
@@ -0,0 +1,63 @@
+package org.molgenis.emx2.io.submission;
+
+import static org.molgenis.emx2.io.submission.SubmissionService.*;
+
+import java.nio.ByteBuffer;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.molgenis.emx2.*;
+
+@Getter
+@Setter
+@Accessors(chain = true)
```

Comment:
what does it do ???

Reply:
RTFM? it creates the .setX().setY() capability.

Reply:
BTW: this discussion on this implementation details is from a business standpoint is not highest priority.